src/sources/audio_base_latency.ts

```diff
@@ -0,0 +1,7 @@
+export default function getAudioContextBaseLatency(): number | undefined {
+  if (!window.AudioContext) {
+    return undefined
+  }
+
+  return new AudioContext().baseLatency
```

Comment:
nit: Could we reuse the AudioContext() from the existing audio signal?

Reply:
It looks that we can't, In existing signal we use `OfflineAudioContext` which doesn't have this property

Let me know if I'm missing something